README.md

```diff
@@ -183,11 +186,90 @@ const App = () => {
       chatBottomContent={customConstants.chatBottomContent}
       messageBottomContent={customConstants.messageBottomContent}
       replacementTextList={customConstants.replacementTextList}
-      customRefreshComponent={customConstants.customRefreshComponent}
     />
   );
 };
 
 export default App;
 
 ```
+
+## Advanced Customization
+```jsx
+import { ChatAiWidget } from "@sendbird/chat-ai-widget";
+import '@sendbird/chat-ai-widget/dist/style.css';
+
+import { SessionHandler } from '@sendbird/chat'
```

Comment:
@sravan-s which one do you recommend? 
```
import SessionHandler from '@sendbird/uikit-react/handlers/SessionHandler'
```
or 
```
import { SessionHandler } from '@sendbird/chat'
```

Reply:
import { SessionHandler } from '@sendbird/chat' > this  is recommended
---  
#1  was a work around for an issue we found in v3 release(I will share ticket)

Reply:
https://sendbird.atlassian.net/browse/CORE-2964

Reply:
Ah I see. Thanks!